src/AddThreepid.js

```diff
@@ -155,7 +159,6 @@ export default class AddThreepid {
      * the request failed.
      */
     async checkEmailLinkClicked() {
-        const identityServerDomain = MatrixClientPeg.get().idBaseUrl.split("://")[1];
```

Comment:
Thanks for this change! It looks like `haveMsisdnToken` should also get the same treatment.